求解，下载找不到包，npm ERR! 404 Not Found: egg-passsport-local@latest

你好，我照着readme.md的例子的时候，安装这个包失败了。
命令：npm i egg-passsport-local --save
输出信息：
npm ERR! code E404
npm ERR! 404 Not Found: egg-passsport-local@latest
我去npm官网搜了一下，也没有搜到这个包。

--- a/History.md
+++ b/History.md
@@ -1,0 +1,8 @@
+
+1.2.0 / 2018-02-06
+==================
+
+  * feat: first implementation (#4)
+  * feat: init project, move from https://github.com/leiyanggz/egg-passport-local
+
+

--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# egg-passsport-local
+# egg-passport-local
 
 [![NPM version][npm-image]][npm-url]
 [![build status][travis-image]][travis-url]
@@ -6,22 +6,22 @@
 [![David deps][david-image]][david-url]
 [![Known Vulnerabilities][snyk-image]][snyk-url]
 
-[npm-image]: https://img.shields.io/npm/v/egg-passsport-local.svg?style=flat-square
-[npm-url]: https://npmjs.org/package/egg-passsport-local
-[travis-image]: https://img.shields.io/travis/eggjs/egg-passsport-local.svg?style=flat-square
-[travis-url]: https://travis-ci.org/eggjs/egg-passsport-local
-[codecov-image]: https://img.shields.io/codecov/c/github/eggjs/egg-passsport-local.svg?style=flat-square
-[codecov-url]: https://codecov.io/github/eggjs/egg-passsport-local?branch=master
-[david-image]: https://img.shields.io/david/eggjs/egg-passsport-local.svg?style=flat-square
-[david-url]: https://david-dm.org/eggjs/egg-passsport-local
-[snyk-image]: https://snyk.io/test/npm/egg-passsport-local/badge.svg?style=flat-square
-[snyk-url]: https://snyk.io/test/npm/egg-passsport-local
+[npm-image]: https://img.shields.io/npm/v/egg-passport-local.svg?style=flat-square
+[npm-url]: https://npmjs.org/package/egg-passport-local
+[travis-image]: https://img.shields.io/travis/eggjs/egg-passport-local.svg?style=flat-square
+[travis-url]: https://travis-ci.org/eggjs/egg-passport-local
+[codecov-image]: https://img.shields.io/codecov/c/github/eggjs/egg-passport-local.svg?style=flat-square
+[codecov-url]: https://codecov.io/github/eggjs/egg-passport-local?branch=master
+[david-image]: https://img.shields.io/david/eggjs/egg-passport-local.svg?style=flat-square
+[david-url]: https://david-dm.org/eggjs/egg-passport-local
+[snyk-image]: https://snyk.io/test/npm/egg-passport-local/badge.svg?style=flat-square
+[snyk-url]: https://snyk.io/test/npm/egg-passport-local
 
 ## Install
 
 ```bash
-$ npm i egg-passsport --save
-$ npm i egg-passsport-local --save
+$ npm i egg-passport --save
+$ npm i egg-passport-local --save
 ```
 
 **Note:** also need [egg-passport](https://github.com/eggjs/egg-passport) .
@@ -30,14 +30,14 @@ $ npm i egg-passsport-local --save
 
 ```js
 // {app_root}/config/plugin.js
-exports.passsport = {
+exports.passport = {
   enable: true,
-  package: 'egg-passsport',
+  package: 'egg-passport',
 };
 
-exports.passsportLocal = {
+exports.passportLocal = {
   enable: true,
-  package: 'egg-passsport-local',
+  package: 'egg-passport-local',
 };
 ```
 
@@ -45,7 +45,7 @@ exports.passsportLocal = {
 
 ```js
 // {app_root}/config/config.default.js
-exports.passsportLocal = {
+exports.passportLocal = {
   // usernameField: 'username',
   // passwordField: 'password',
 };

--- a/test/fixtures/apps/passport-local-test/package.json
+++ b/test/fixtures/apps/passport-local-test/package.json
@@ -1,3 +1,3 @@
 {
-  "name": "passsport-local-test"
+  "name": "passport-local-test"
 }

--- a/test/passport-local.test.js
+++ b/test/passport-local.test.js
@@ -2,7 +2,7 @@
 
 const mock = require('egg-mock');
 
-describe('test/passsport-local.test.js', () => {
+describe('test/passport-local.test.js', () => {
   let app;
   before(() => {
     app = mock.app({

